### PR TITLE
dev_ag1

### PR DESCRIPTION
**Pull request by anadi**
Please review my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # mylog
-anadis mylog project
+anadis mylog project.
+
+## Goals
```
Comment (anadi-github) [inline README.md:4]: This is an inline comment

**Review by anadi-github — COMMENTED** [review]
AG review of pull request #1 is done

**Comment by anadi-github** [comment]
@anadi please see my inline comment

File: README.md (hunk 1 above)
Comment (anadi-teemup) [reply README.md:4]: Looks OK to me

**Comment by anadi-teemup** [comment]
reopeing to merge

**Pull request merged**